software/nanolyse/meta.yml

```diff
@@ -0,0 +1,40 @@
+name: nanolyse
+description: DNA contaminant removal using NanoLyse
+keywords:
+    - contaminant_removal
+tools:
+    - nanolyse:
+        description: |
+            DNA contaminant removal using NanoLyse
+        homepage: https://github.com/wdecoster/nanolyse
+        documentation: https://github.com/wdecoster/nanolyse#nanolyse
+input:
+    - meta:
+        type: map
+        description: |
+            Groovy Map containing sample information
+            e.g. [ id:'test', single_end:false ]
+    - fastq:
+        type: file
+        description: |
+            Basecalled reads in FASTQ.GZ format
+    - nanolyse_fasta:
+        type: file
+        description: |
+            A reference fasta file against which to filter.
```

Comment:
would be nice to have the pattern of the files here :)

Reply:
I have added the patterns